cmd/update-main.go

```diff
@@ -100,7 +100,7 @@ func GetCurrentReleaseTime() (releaseTime time.Time, err error) {
 
 func isDocker(cgroupFile string) (bool, error) {
 	cgroup, err := ioutil.ReadFile(cgroupFile)
-	if os.IsNotExist(err) {
+	if os.IsNotExist(err) || os.IsPermission(err) {
```

Comment:
I don't think we can infer that its a setup issue @balamurugana by the condition that its not readable. There is no set document that indicates that this file should be readable by everyone, it is readable perhaps is purely incidental and has nothing to do with it. 

Neither its a requirement for Minio server that this file should be readable, it is a guess work at best and works well for the most part because docker makes this file available with some `docker` strings in it.  We will be changing all this code to make a proper API call to know we are indeed inside docker.

Let me not ignore the permission error for now until we find a better solution.